sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample3_BackRestoreResume.md

```diff
@@ -0,0 +1,46 @@
+# Checking the status of a previously started full key backup and restore
+
+This sample demonstrates how to a check the status and get the result of a previously started full key backup and restore in Azure Key Vault.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/README.md) for links and instructions.
+
+## Checking status of a full key backup
```

Comment:
```suggestion
## Checking status of a full key backup operation
```

This may translate better, if ever we do that. We assume operation, but may not translate properly otherwise.